frontend/projects/upgrade/src/app/core/experiments/store/experiments.effects.ts

```diff
@@ -486,6 +487,7 @@ export class ExperimentEffects {
                 this.download('Experiments.zip', content, true);
               });
             } else {
+              data[0].state = EXPERIMENT_STATE.INACTIVE;
```

Comment:
i think this approach works fine, you'll also want to do it in the 'bulk export' in the if clause.

```
data.forEach((experiment, index) => {
                experiment.state = EXPERIMENT_STATE.INACTIVE;
                zip.file(experiment.name + ' (File ' + (index + 1) + ').json', JSON.stringify(experiment));
              });
              zip.generateAsync({ type: 'base64' }).then((content) => {
                this.download('Experiments.zip', content, true);
              });
```